experiments/train_dt4rec.py

```diff
@@ -0,0 +1,60 @@
+from rs_datasets import MovieLens
+
+from replay.metrics import MAP, MRR, NDCG, Coverage, HitRate, Surprisal
+from replay.metrics.experiment import Experiment
+from replay.models import DT4Rec
+from replay.preprocessing.data_preparator import DataPreparator, Indexer
```

Comment:
Please rebase, DataPreparator and Indexer in experimental part of the library now. 
So, please rewrite the algorithm to support Dataset or move it to experimental also